index.html

```diff
@@ -148,6 +148,43 @@ <h1>Multi-Post Stories</h1>
  
             </article>
         </section>
+        <section class="contact-form">
+            <h3>CONTACT US</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p> 
+            <form class="form" action="https://formspree.io/f/xoqrbwdq" method="POST">
+                <div>
+                    <label for="text">
+                        <input type="text" id="text" placeholder="Text input" required minlength="6" maxlength="30">

```

Comment:
- [ ] Kindly add the correct `placeholder` text for this which is `Full Name`, It was just a mistake in the Figma template.